README.md

```diff
@@ -53,3 +53,26 @@ openshift-install destroy cluster
 Note that you almost certainly also want to clean up the installer state files too, including `auth/`, `terraform.tfstate`, etc.
 The best thing to do is always pass the `--dir` argument to `install` and `destroy`.
 And if you want to reinstall from scratch, `rm -rf` the asset directory beforehand.
+
+
+### Troubleshouting installation
```

Comment:
This doesn't belong in the project README.  We have a platform-agnostic troubleshooting doc in `docs/user/troubleshooting.md` (already linked from the project README), and AWS-specific troubleshooting docs should go in a new `docs/user/aws/troubleshooting.md` (like our existing `docs/user/openstack/troubleshooting.md`).

Reply:
I can send it there if this is not best place ... ?